server/routes/visitJourney/dateAndTime.ts

```diff
@@ -20,37 +20,36 @@ export default class DateAndTime {
     const { prisonId } = req.session.selectedEstablishment
     const { visitSessionData } = req.session
 
-    const warningMessages: { id: string; message: string }[] = []
+    const messages: MoJAlert[] = req.flash('messages')
```

Comment:
Using `messages` here for any messages that have come via `req.flash()` or are set in this controller. Previously multiple different variables were used to pass different kinds of message to the view.